docs/content/reboot.md

```diff
@@ -301,3 +301,13 @@ HTML5 adds [a new global attribute named `[hidden]`](https://developer.mozilla.o
 {% endcallout %}
 
 To merely toggle the visibility of an element, meaning its `display` is not modified and the element can still affect the flow of the document, use [the `.invisible` class]({{ site.baseurl }}/components/utilities/#invisible-content) instead.
+
+## Touch responsiveness optimization
```

Comment:
"response" rather than "responsiveness" perhaps? or simply "Touch optimization" (though this may be wrongly be interpreted as a much larger topic, like touch-optimized carousels etc)?